1.7/administration/high-availability/mult-zone.md

```diff
@@ -0,0 +1,41 @@
+---
+post_title: Multiple Zones
+menu_order: 1
+---
+
+Here are recommendations for specific multi-AZ configurations.
+
+**Important:** A typical DC/OS cluster has all master and agent nodes in the same zone. The cost of having masters spread across zones usually outweigh the benefits. None of the following multi-AZ setups have been explicitly tested or verified by Mesosphere.
+
+# Single-zone masters and cross-zone agents within a region
+All DC/OS masters are present in a single zone, but the agents can span multiple zones within that region.
+
+#### Recommendations
+
+
+- If you are using a rack, you should distribute masters across the racks.
+- All agents within a zone should be tagged with an attribute (e.g., `zone:us-east-1a`) to easily constrain apps and services to specific zones.
+- Any DC/OS service schedulers should be scheduled to the same zone as masters (e.g., by using constraints).
```

Comment:
My comment didn't end up in the right spot. I was referring to the item below, "...ensure that apps do not depend on zone-level resources..."